packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -1083,7 +1121,17 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
 		});
 	}
 
-	public generateAttachMessage(): IAttachMessage {
+	/**
+	 * @see FluidDataStoreContext.getAttachData
+	 */
+	public getAttachData(
```

Comment:
It would be cleaner to have 2 separate functions - getAttachSummary and getAttachGCData. It keeps them summary and GC separate and you won't need the `includeGCData` param here.

Reply:
I was debating - It's minimal but I like avoiding doubling up on those asserts.  Felt worth it - and the function is factored so it will be trivial to pull apart later if needed.

Reply:
(Decided to Won't Fix this)